### ISSUE TITLE
[A11Y] Fix Website Accessibility Issues

### ISSUE THREAD

**Issue by oleksis** [BlackPythonDevs/blackpythondevs.github.io#275]
Fix some issues for Accessibility :

![A10Y-Issues_13-3-2024_22656_blackpythondevs com](https://github.com/BlackPythonDevs/blackpythondevs.github.io/assets/44526468/9863d94e-a46a-4ccf-a386-6c60169bb82b)
